otherlibs/unix/unixsupport.h

```diff
@@ -123,7 +111,37 @@ extern int caml_win32_set_inherit(HANDLE fd, BOOL inherit);
 #else
 extern void caml_unix_set_cloexec(int fd, char * cmdname, value arg);
 extern void caml_unix_clear_cloexec(int fd, char * cmdname, value arg);
-#endif
+#endif /* _WIN32 */
+
+/* Compatibility definitions for the pre-5.0 names of these functions */
+#ifndef CAML_BUILDING_UNIX
+#ifdef _WIN32
+#define win_alloc_handle caml_win32_alloc_handle
+#define win_alloc_socket caml_win32_alloc_socket
+#define win_CRT_fd_of_filedescr caml_win32_CRT_fd_of_filedescr
+#define win32_maperr caml_win32_maperr
```

Comment:
The `win32_maperr` macri is also defined at line 78.

Reply:
Good catch - in fact, that macro not being guarded on L78 in the same way as the others means that there was a renaming missing in `socket_win32.c` (presumably a rebase artefact during the original PR)